example/app/data/DemoDataSource/plugins/view_selector/car_garage/blueprints/Owner.blueprint.json

```diff
@@ -0,0 +1,28 @@
+{
+  "type": "CORE:Blueprint",
+  "name": "OwnerBlueprint",
+  "extends": ["CORE:NamedEntity"],
+  "attributes": [
+    {
+      "attributeType": "string",
+      "type": "CORE:BlueprintAttribute",
+      "name": "name",
+      "label": "Name of Owner",
+      "optional": false
+    },
```

Comment:
```suggestion

```

name in the attributes list is not needed since it is included in NamedEntity which OwnerBlueprint extends from

Reply:
But then I can´t change the label, can I?

Reply:
oh right, that is not possible. 
The name can  be kept in the attributes list then 👍